src/modules/evaluateExpression/evaluateExpression.ts

```diff
@@ -1,45 +1,9 @@
-interface IParameters {
-  [key: string]: any
-  connection?: IConnection
-}
-
-interface IQueryNode {
-  value?: string | number | boolean | object
-  type?: NodeType
-  operator?: Operator
-  children?: Array<IQueryNode>
-}
-
-interface IConnection {
-  user?: string
-  host?: string
-  database?: string
-  password?: string
-  port?: number
-  query?: any
-}
-
-type NodeType = 'string' | 'number' | 'boolean' | 'array'
-
-type Operator =
-  | 'AND'
-  | 'OR'
-  | 'CONCAT'
-  | '='
-  | '!= '
-  | '+'
-  | 'REGEX'
-  | 'objectProperties'
-  | 'pgSQL'
-  | 'graphQL'
-
-const defaultParameters: IParameters = {
-  connection: {},
-}
+import { IQueryNode, IParameters } from '../../types'
```

Comment:
Can we NOT put the types for this module into the project `types` file, since we want to extract this as its own package at some point? Maybe make a separate `types` file in this modules folder?

Reply:
Sure. Indeed. I was considering putting every type inside this file, but you are right this will be divided in different projects/packages soon. I'll move to a separated file inside the same module : )